### ISSUE TITLE
Video Source Format

### ISSUE THREAD

**Issue by SterlingButters** [freshwuzhere/react-player-dash#1]
What are acceptable video “sources”? Would it be possible to feed cv2 video to this player?

**Comment by freshwuzhere** [comment]
It can handle all sources from react player.  

https://www.npmjs.com/package/react-player

It’s a big list but not cv2 (I don’t think).  I have only had success with web served video files (local js web server is an option) - not local files which I think is a dash shortcoming.  

I am not sure internally what a CV2 files is so not aware of a quick file fix. You may need to export the file as another format or maybe stream it. 

Ian Burns
AUS   +61 466 421 293
USA   +1 415 309 2298

**Comment by SterlingButters** [comment]
Thanks for your response. I have found some things that may work. I may post back if I figure something out.

**Comment by freshwuzhere** [comment]
Great -LMK. 

to save anything from OpenCV you need to pick your file type and it’s easy from there.  But - if you want to do it ‘in memory’ it’s hard to connect python memory to java script memory AFAIK. 

Ian Burns
AUS   +61 466 421 293
USA   +1 415 309 2298

**Comment by SterlingButters** [comment]
Yes I was running into memory translation issue not just with video. I was looking into Redis sever for that originally. Now I am wondering about creating custom react.js component that I can port into Plotly’s Dash for Python. This of course assumes I know anything about react.js which I don’t yet haha so it may be awhile before I post back.
